proto/internalapi/central/compliance_operator.proto

```diff
@@ -265,3 +265,15 @@ message ComplianceOperatorSuiteV2 {
   reserved 3;
   ComplianceOperatorStatus status = 4;
 }
+
+message ComplianceOperatorRemediationV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Remediation Name,hidden"
+  // TODO: Name or ID?
+  string compliance_check_result_name = 3; // @gotags: search:"Compliance Check Result Name,hidden"
+  bool apply = 4; // @gotags: search:"Compliance Remediation Applied,hidden"
+  string current = 5;
+  string outdated = 6;
+  string type = 7;
+  string cluster_id = 8;
```

Comment:
We don't need the `cluster_id` here.  I'm not even sure if it is available.  In general when this message gets sent to central and processed by the pipeline, the pipeline knows which cluster sent it.  So the pattern is in the pipeline to set the `cluster_id` in the storage object based on the processing of the pipeline message.

Reply:
Some had the cluster_id set, some protos don't. Some have `reversed = ...` instead. Can I remove it or use `reversed`?

Reply:
Imho reversed is not necessary, the conversion is manual anyway.

Reply:
Removed it

Reply:
we used `reserved` to maintain backwards compatibility.  So that is only needed if you remove something that was previously included in a release.  So in this case you are good to remove it.